tests/test_solvermode/test_boundaries.py

```diff
@@ -40,7 +41,7 @@ def test_boundaries_elbow(load_mixing_elbow_mesh):
         "velocity_spec": "Magnitude, Normal to Boundary",
         "frame_of_reference": "Absolute",
         "vmag": D(0.4),
-        "p_sup": D(0),
+        api_names(solver_session).initial_gauge_pressure: D(0),
         "t": D(293.15),
```

Comment:
@akulshre-qa, this was a key mismatch issue basically, thank you @seanpearsonuk , for helping me resolve it effectively.

Reply:
@akulshre-qa, regarding the "test_boundaries_periodic", there is a confusion on who is maintaining this test. It contains a huge json file that needs to be fixed and is tricky to compare as well. So, if we don't need this test maybe it can be dropped off? @seanpearsonuk, may be you can add something more on what to do with this test?
Thank you :)

Reply:
Yes, we won't invest time in that. If the files won't be maintained they can be removed instead.